app/src/main/res/layout/activity_create_account.xml

```diff
@@ -6,12 +6,15 @@
     android:layout_height="match_parent"
     tools:context=".CreateAccountActivity">
 
+    <!--    Todo: remove explicit set textColor black-->
```

Comment:
Any specific ideas how?

Reply:
Also why is it a problem to explicitly set it to black?

Reply:
Thanks for noticing. 
We could set the text color to black in a theme defined in styles.xml and apply it to login and create account activity
This would allow easier to maintain.